npm-shrinkwrap.json

```diff
@@ -5581,7 +5581,7 @@
         },
```

Comment:
Nothing relevant to this PR here (we can keep it, just noting that there are no package ref changes or similar)